### PR TITLE
Add gas limit options

### PR DESCRIPTION
**Pull request by smk762**
Adds notes for `gas_limit` param object as detailed in https://github.com/KomodoPlatform/komodo-defi-framework/pull/2137

### CODE REVIEW

File: src/pages/komodo-defi-framework/tutorials/listing-a-new-coin/index.mdx [hunk 1]
```diff
@@ -125,13 +125,28 @@ The token's contract **must** have `approve` and `transferFrom` methods. Additio
...
+| ----------------------------------------- | ------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| chain\_id                                 | integer | To find the EVM chain ID, see [Chainlist](https://chainlist.org/)                                                                                                                                                                                                   |
+| gas\_limit                                | object  | Optional, required for tokens only. Defines more precise gas prices for swap or transaction operations, useful for tokens called ovcer proxy contracts which often require more than the default gas. See the `Gas Limit Options` table below for more information. |
```
Comment (shamardy) [inline src/pages/komodo-defi-framework/tutorials/listing-a-new-coin/index.mdx:131]: > useful for tokens called ovcer proxy contracts which often require more than the default gas.

It's used for the reverse of this actually, we use default limits that allow tokens that use proxy contracts but we can use these parameters to lower limits for other tokens if needed. @dimxy please confirm this and also give this docs PR a review.

Comment (dimxy) [reply src/pages/komodo-defi-framework/tutorials/listing-a-new-coin/index.mdx:131]: Yes all gas_limit params are optional and internally set to default values (which maybe too high for certain tokens, so could be tuned for new or existing evm tokens)

**Review by dimxy — APPROVED** [review]
lgtm

**Review by cipig — APPROVED** [review]

**Comment by smk762** [comment]
@gcharang This one is ready to merge on your approval

**Pull request merged**